cg/test/management/stats.api.spec.js

```diff
@@ -0,0 +1,107 @@
+jest.mock('../../src/utils/blockchain/web3-provider-factory');
+
+const winston = require('winston');
+
+const { initResources, fetch, closeResources } = require('../utils');
+const { deployContract } = require('../blockchain-setup');
+const { db } = require('../../src/db');
+const { managementJWT, subjectJWT } = require('../../src/utils/jwt');
+const { setProcessors } = require('../../src/utils/blockchain');
```

Comment:
Just a reminder that setProcessors is meant to facilitate tests and should not be used in the actual API